src/components/nodes-table/nodes-table.js

```diff
@@ -0,0 +1,60 @@
+import { NodeCollection } from '@/modules/NodeCollection'
+
+export default {
+  props: {
+    nodes: Array
+  },
+  data: () => ({
+    selectedNode: undefined,
+    filters: {},
+    headers: [
+      { text: 'ID', value: 'node_id' },
+      { text: 'Type', value: 'type' },
+      { text: 'Product', value: 'product' },
+      { text: 'Name', value: 'name' },
+      { text: 'Location', value: 'loc' },
+      { text: 'Secure', value: 'secure' },
+      { text: 'Status', value: 'status' },
+      { text: 'Last Active', value: 'lastActive' }
+    ]
+  }),
+  methods: {
+    nodeSelected (node) {
+      this.selectedNode = node
+      this.$emit('node-selected', { node })
+    },
+    productName (node) {
+      const manufacturer = node.manufacturer ? ` (${node.manufacturer})` : ''
+      return node.ready ? `${node.product}${manufacturer}` : ''
+    }
+  },
+  computed: {
+    nodeCollection () {
+      return new NodeCollection(this.nodes)
+    },
+    filteredNodes () {
```

Comment:
Why don't use vuetify custom filters? https://vuetifyjs.com/en/components/data-tables/

Reply:
`custom-filter` is applied to all the items with a single value provided by a search box. Using the node collection we can filter the different columns separately, so we can filter out the dead nodes and search a product name for a specific type, as an example.